src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/main.js

```diff
@@ -124,6 +117,20 @@ define([
         },
 
         /**
+         * go to accounts list
+         * @param account
+         * @param noReload
+         */
+        goToList: function(account, noReload) {
```

Comment:
I would call this method `navigateToList`, `goTo` sounds like a bad programming construct :wink: